templates/errors/no-cookie.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layouts/_base.html' %}
+
+{% set survey_title = _("Session expired") ~ " - " ~ _("Census 2021") %}
```

Comment:
The page title should match the h1, so this should be `Sorry there is a problem | Census 2021`

Reply:
So as discussed checked with Laura and this will be updated to `Page is not available`